osqp.m

```diff
@@ -554,7 +554,7 @@ function codegen(this, target_dir, varargin)
             % Rename the mex file
             old_mexfile = ['emosqp_mex.', mexext];
             new_mexfile = [p.Results.mexname, '.', mexext];
-            movefile(old_mexfile, new_mexfile);
+            movefile(old_mexfile, new_mexfile, 'f');
```

Comment:
Will moving the file keep the read-only attribute? Maybe we should add a call to `fileattrib( old_mexfile, '+w' )` to make it writtable again.

Reply:
Actually the issue goes deeper than this on Windows, as I discovered. The Desktop in Windows 10 (where I was testing this on), is read-only for some reason, which causes applications that try to directly write to it choke on permissions. Running `cmake` as a process from within MATLAB works fine, but the `movefile` by MATLAB fails. Makes no sense from a security perspective.

Making folders non-readonly is not trivial on Windows anymore (involves going deep inside the Control Panel), and running osqp-matlab on a "regular" non-desktop folder works just fine. Due to this being a potential rabbit hole, let me close this issue.